tests/SprykerSdkTest/Integrator/IntegratorFacadeGeneralTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdkTest\Integrator;
+
+class IntegratorFacadeGeneralTest extends AbstractIntegratorTestCase
+{
+    /**
+     * @var string
+     */
+    protected const MANIFESTS_DIR_PATH = 'general/manifests/src';
+
+    /**
+     * @var string
+     */
+    protected const ZIP_PATH = '_data/archive.zip';
+
+    /**
+     * Rewrite the path to mock project
+     *
+     * @return string
+     */
+    public function getProjectMockOriginalPath(): string
```

Comment:
this method should be protected if it is not a test method or isn't a data provider.

Reply:
Yes, I missed that. I changed the visibility of the parent methods, it really has no sense to make them public